dbgap_extract.py

```diff
@@ -176,7 +176,7 @@ def _get_flattened_sra_data_details_from_xml_sample(sample):
         for stat in sra_datas:
             stat_dict = stat.attrib
             stats_as_string = ""
-            for key in stat_dict:
+            for key in {k: stat_dict[k] for k in sorted(stat_dict)}:
```

Comment:
i think this line undoes itself -- the sorted(stat_dict) does indeed sort the keys, but once those keys are wrapped in a new dictionary, the ordering is undone